sympy/concrete/summations.py

```diff
@@ -483,7 +483,10 @@ def is_convergent(self):
             return S.false
 
         ### ------------- integral test -------------- ###
-        if is_decreasing(sequence_term, interval):
+        inequality = Derivative(sequence_term).doit() <= 0
```

Comment:
At lines 386-388: 
```
        if len(sequence_term.free_symbols) > 1:
            raise NotImplementedError("convergence checking for more than one symbol "
                                      "containing series is not handled")

```
If it is multivariate, the method should stop here. I modify according to your suggestion.